### ISSUE TITLE
html generation fault

### ISSUE THREAD

**Issue by phreed** [CategoricalData/fql#110]
The HTML for AQL generates but the pretty-printing of the AQL statements has some issues.
For example the "Tutorial TSP" example,
the following AQL ...
```aql
schema sT = literal : TypeSide {
    entities
        Task
    attributes
        TName : Task -> Varchar
}
```
..is rendered...
```html
<pre>schema sT = 	entities
		Task
	attributes
		TName : Task -&gt; Varchar
</pre>
```

**Comment by wisnesky** [comment]
Pretty printing is basically my best ad hoc effort; it’s guaranteed to be bad in some situations.

Can we generate a pretty printer from ANTLR?  If so we could fix the HTML output and also get an AQL code formatter.

**Comment by phreed** [comment]
Yes, pretty-printing from the ANTLR concrete syntax tree is one of the easier things to do.
